csrankings.csv

```diff
@@ -306,6 +306,7 @@ Ailamaki Natassa,EPFL,https://people.epfl.ch/anastasia.ailamaki,YdrkeNwAAAAJ
 Aiman Erbad,Qatar University,http://www.qu.edu.qa/engineering/computer/faculty/aiman.php,6Pmbv8IAAAAJ
 Aiman Hanna,Concordia University,http://aimanhanna.com/concordia,NOSCHOLARPAGE
 Aimin Hao,Beihang University,http://scse.buaa.edu.cn/info/1078/2654.htm,NOSCHOLARPAGE
+Aishwarya Agrawal,Georgia Institute of Technology,https://www.cc.gatech.edu/grads/a/aagrawal307,pLurJcoAAAAJ

```

Comment:
Please correct the affiliation to McGill University.

Reply:
(or Montreal, whichever it is)

Reply:
Hello Emery , for some reason , Im not able to repush the pull request, I'm new at this...!

The correct affiliation for Aishwarya Agrawal it's University of Montréal